pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -20,10 +20,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var BundleKind = "Bundle"
+const (
+	BundleKind = "Bundle"
 
-var BundleLabelKey = "trust.cert-manager.io/bundle"
-var BundleHashAnnotationKey = "trust.cert-manager.io/hash"
+	BundleLabelKey                   = "trust.cert-manager.io/bundle"
+	BundleHashAnnotationKey          = "trust.cert-manager.io/hash"
+	BundleJksPasswdHashAnnotation    = "trust.cert-manager.io/jks-pwd-hash"    // # nosec G101
+	BundlePkcs12PasswdHashAnnotation = "trust.cert-manager.io/pksc12-pwd-hash" // # nosec G101
```

Comment:
```suggestion
	BundleJksPasswdHashAnnotationKey    = "trust.cert-manager.io/jks-pwd-hash"    // #nosec G101
	BundlePkcs12PasswdHashAnnotationKey = "trust.cert-manager.io/pksc12-pwd-hash" // #nosec G101
```

It seems like gosec does not understand the suppressions. Looking at other suppressions in this project, I think the space matters. Also suggesting to follow the `(...)Key` pattern established just above.